module/metrics/noop.go

```diff
@@ -118,9 +121,10 @@ func (nc *NoopCollector) ReadDuration(duration time.Duration)
 func (nc *NoopCollector) ReadDurationPerItem(duration time.Duration)                             {}
 func (nc *NoopCollector) ExecutionCollectionRequestSent()                                        {}
 func (nc *NoopCollector) ExecutionCollectionRequestRetried()                                     {}
-func (nc *NoopCollector) TransactionParsed(dur time.Duration)                                    {}
-func (nc *NoopCollector) TransactionChecked(dur time.Duration)                                   {}
-func (nc *NoopCollector) TransactionInterpreted(dur time.Duration)                               {}
```

Comment:
these were renamed to not conflict with Transaction end-to-end lifetime metrics such as time to finalized, ...